impl/maven-impl/src/main/java/org/apache/maven/impl/DefaultDependencyResolver.java

```diff
@@ -67,6 +67,21 @@
 @Named
 @Singleton
 public class DefaultDependencyResolver implements DependencyResolver {
+    /**
```

Comment:
```suggestion

    /**
```
cs can/must fail this.

Reply:
Done.